standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestTablesGetExists.java

```diff
@@ -271,6 +271,20 @@ public void testGetTables() throws Exception {
     Assert.assertEquals("Found functions size", 1, tables.size());
     Assert.assertTrue("Comparing tablenames", tables.contains(testTables[6].getTableName()));
 
+    // Find tables by using the wildcard sign "*"
+    tables = client.getTables(DEFAULT_DATABASE, "*");
```

Comment:
I guess we need to test searching the tables with database '*', `getTables(DEFAULT_DATABASE, "*")` means the table pattern is '*', it's not reflect the changes.

Reply:
How about testing `client.getTables("def*", "*")` and `client.getTables("*no_such_database*", "*")` the first argument for the database is a pattern.